Wikipedia/Code/CIDetector+WMFFaceDetection.m

```diff
@@ -1,27 +1,21 @@
-//
-//  CIDetector+WMFFaceDetection.m
-//  Wikipedia
-//
-//  Created by Brian Gerstle on 7/20/15.
-//  Copyright (c) 2015 Wikimedia Foundation. All rights reserved.
-//
-
 #import "CIDetector+WMFFaceDetection.h"
 #import "Wikipedia-Swift.h"
 
 #import <BlocksKit/BlocksKit.h>
 #import "CIContext+WMFImageProcessing.h"
 #import "UIImage+WMFImageProcessing.h"
 
+NSString* const WMFFaceDetectionErrorDomain = @"org.wikimedia.face-detection-error";
+
 @implementation CIDetector (WMFFaceDetection)
 
-+ (instancetype)wmf_sharedBackgroundFaceDetector {
++ (instancetype)wmf_sharedFaceDetector {
     static CIDetector* defaultFaceDetector;
     static dispatch_once_t onceToken;
     dispatch_once(&onceToken, ^{
         defaultFaceDetector = [CIDetector detectorOfType:CIDetectorTypeFace
-                                                 context:[CIContext wmf_sharedBackgroundContext]
-                                                 options:nil];
+                                                 context:[CIContext wmf_sharedContext]
+                                                 options:@{CIDetectorAccuracy: CIDetectorAccuracyLow}];
```

Comment:
Did we verify this being OK for our images… any major differences in face detection?